lib/ctype.c

```diff
@@ -1,9 +1,9 @@
 /*
  * Phoenix-RTOS
  *
- * phoenix-rtos loader
+ * Operating system loader
  *
- * ctype - code derived from libphoenix
+ * Character type
```

Comment:
The comment that code is derived from libphoenix should be left to know its origin.
I think it will be important if the common part is shared between plo, kernel and libphoenix.

Reply:
Restored libphoenix origin info.